### PR TITLE
A11y: Don't init captions if they are disabled; improved settings UI

### PR DESCRIPTION
**Pull request by kevinfoley**
- If none of the caption modes are enabled when TFE starts, we now skip the caption initialization process. Captions will be initialized if the user navigates to the Accessibility menu.
- Improved UI layout for Accessibility settings menu; fixed issue where the example caption was obscuring some settings on small screens.
- Restored functionality that adjusts the maximum number of caption lines during cutscenes based on the selected font size.

### CODE REVIEW

**Comment by kevinfoley** [comment]
@mlauss2 I can't believe I missed that typo! Thanks for pointing it out. I did add a divider and spacing as part of the UI improvements.

**Review by luciusDXL — APPROVED** [review]

File: TheForceEngine/TFE_A11y/accessibility.cpp [hunk 1]
```diff
@@ -146,7 +154,9 @@ namespace TFE_A11Y  // a11y is industry slang for accessibility
 
 		// Try to load the previously selected font.
 		string lastFontPath = TFE_Settings::getA11ySettings()->lastFontPath;
-		tryLoadFont(lastFontPath, false);
+
+		if (lastFontPath, ImGui::GetIO().Fonts->Locked)	{ setPendingFont(lastFontPath);	}
```
Comment (mlauss2) [inline TheForceEngine/TFE_A11y/accessibility.cpp:158]: that "lastFontPath," is that a copy-paste leftover?

Comment (kevinfoley) [reply TheForceEngine/TFE_A11y/accessibility.cpp:158]: Looks like that was supposed to be a `&&` instead of a comma.

Comment (mlauss2) [reply TheForceEngine/TFE_A11y/accessibility.cpp:158]: like this?  (!lastFontPath.empty() && ImGui::GetIO.... ) ?

**Pull request merged**